### PR TITLE
r9m receiver

### PR DESCRIPTION
**Pull request by jlpoltrack**
@brad112358  @olliw42 
Initial notes on how to use R9M as receiver.  Let me know if there's additional pieces that you think should be included.

### CODE REVIEW

File: docs/FRSKY_R9.md [hunk 1]
```diff
@@ -67,7 +67,24 @@ A description for installing a fan along with documentation for 3D printed case
 
 ### As Receiver ###
 
-The R9M module can also be flashed with firmware which allows it to be used as mLRS receiver. This yields a receiver working at high transmit power for longer range applications. For details please search the rcgroups thread and ask on rcgroups or discord if you have questions.
+The R9M module can also be flashed with firmware which allows it to be used as mLRS receiver. This yields a receiver working at high transmit power for longer range applications. 
+
+Connections are as follows:
+
+4-Pin Header:
```
Comment (olliw42) [inline docs/FRSKY_R9.md:74]: shiuldn't this be alreay in the R9M tx section?

File: docs/FRSKY_R9.md [hunk 2]
```diff
@@ -67,7 +67,24 @@ A description for installing a fan along with documentation for 3D printed case
...
+- Inverted serial Rx
+- Inverted serial Tx
+- 5 Volts
+- GND
+
+Note: As the serial connections are inverted, these will have to be accounted for when connecting to the flight controller.  One solution is to use an inverter cable as mentioned above.  Alternatively, F7 and H7 based flight controllers have built-in inverters which can be enabled with the SERIALx_OPTIONS parameter.
```
Comment (olliw42) [inline docs/FRSKY_R9.md:80]: "... which, e.g., for AduPilot can be ..."

File: docs/FRSKY_R9.md (hunk 1 above)
Comment (jlpoltrack) [reply docs/FRSKY_R9.md:74]: Not at the moment - does it make sense to add a 'Connections' section for the Tx?

File: docs/FRSKY_R9.md (hunk 2 above)
Comment (jlpoltrack) [reply docs/FRSKY_R9.md]: Good point, made this ArduPilot specific.

File: docs/FRSKY_R9.md (hunk 1 above)
Comment (jlpoltrack) [reply docs/FRSKY_R9.md:87]: Added

Comment (olliw42) [reply docs/FRSKY_R9.md:87]: that's really just taste and nitpicking (jo, in certain matters I am nitpicky) ... for the R9M tx module section, the "Connections" stand a bit loose and come a bit unexpected. What about moving them to after the 3 bullets and before the tx versions sub chapter, and to make "Connections" also a sub chapter, i.e. "### Connections ###". Just a thought...

Comment (jlpoltrack) [reply docs/FRSKY_R9.md:87]: Easy enough, other pages (like Seeed) do Connections first but can see how it makes sense here.

Comment (olliw42) [reply docs/FRSKY_R9.md:87]: it's just a thought stream uniltered by my brain ... may not make sense ...

Comment (olliw42) [reply docs/FRSKY_R9.md:87]: I somehow miss a bit an intro sentence. I liked the original one. Maybe "The R9M module can also be used as mLRS receiver, when flashed with the corresponding firmware. This yields a receiver with high transmit power for long range applications."

File: docs/FRSKY_R9.md [hunk 3]
```diff
@@ -67,8 +82,22 @@ A description for installing a fan along with documentation for 3D printed case
...
-The R9M module can also be flashed with firmware which allows it to be used as mLRS receiver. This yields a receiver working at high transmit power for longer range applications. For details please search the rcgroups thread and ask on rcgroups or discord if you have questions.
+Connections:
+
+4-Pin Header:
+- Inverted serial Rx
+- Inverted serial Tx
+- 5 Volts
+- GND
+
+Note: As the serial connections are inverted, these will have to be accounted for when connecting to the flight controller.  One solution is to use an inverter cable as mentioned above.  Alternatively, ArduPilot F7 and H7 based flight controllers have built-in inverters which can be enabled using with the SERIALx_OPTIONS parameter.
```
[the remark this answers is not in the document]
Comment (olliw42) [reply docs/FRSKY_R9.md]: ah ... you see ... German English ...

**Comment by olliw42** [comment]
:)
THX!
lgtm

**Pull request merged**